### DISCUSSION TITLE
The deFooocus folder is missing on the zip file

### DISCUSSION THREAD

**Discussion by shahriyardx** [ehristoforu/DeFooocus#16] [General]
<img width="522" alt="Screenshot 2024-05-31 025140" src="https://github.com/ehristoforu/DeFooocus/assets/40076722/0b4e0a64-7d07-46ac-96c4-db39455c4c08">

**Comment by ehristoforu** [comment]
Run install.bat and the folder will appear, then run.bat you need.
